ios/Classes/TiAirshipModule.m

```diff
@@ -118,6 +118,28 @@ - (void)setUserNotificationsEnabled:(id)value {
     [[UAirship push] updateRegistration];
 }
 
+- (id)isAutoBadgeEnabled {
+    return NUMBOOL([UAirship push].autobadgeEnabled);
+}
+
+- (void)setIsAutoBadgeEnabled:(id)args {
+    [UAirship push].autobadgeEnabled = [TiUtils boolValue:args
+                                                      def:[UAirship push].autobadgeEnabled];
```

Comment:
what are the conditions in which args could be nil? I'm wondering if maybe we want to do a similar thing in setUserNotificationsEnabled, which currently has YES as the default value.